app/state_machines/efile_submission_state_machine.rb

```diff
@@ -133,11 +135,16 @@ class EfileSubmissionStateMachine
   after_transition(to: :rejected, after_commit: true) do |submission, transition|
     AfterTransitionTasksForRejectedReturnJob.perform_later(submission, transition)
     if submission.is_for_state_filing?
-      StateFile::AfterTransitionMessagingService.new(submission).send_efile_submission_rejected_message
       EfileSubmissionStateMachine.send_mixpanel_event(submission, "state_file_efile_return_rejected")
     end
   end
 
+  before_transition(to: :notified_of_rejection) do |submission, transition|
```

Comment:
since transition is never used in this block
```suggestion
  before_transition(to: :notified_of_rejection) do |submission|
```